### ISSUE TITLE
DS4 Touch not working

### ISSUE THREAD

**Issue by Riff-Skippelton** [JibbSmart/JoyShockMapper#152]
The software doesn't seem to register touchpad input on my ds4 at all. Unfortunately, as great as the software is, it's useless to me unless I can map the touchpad to gyro off to be able to quickly re-center aim.

**Comment by ShogunPatch** [comment]
Yes, I too am having trouble getting the touchpad to work.  \in the meantime I've just mapped a chorded press to temporarily disable gyro when I need to.

**Comment by JibbSmart** [comment]
Touchpad touch wasn't a feature until the version 3 release last week. Please check that out here: https://github.com/Electronicks/JoyShockMapper
